src/couch/src/couch_httpd_auth.erl

```diff
@@ -305,126 +368,175 @@ cookie_auth_header(#httpd{user_ctx=#user_ctx{name=User}, auth={Secret, true}}=Re
             % clear the session
-            Cookie = mochiweb_cookies:cookie("AuthSession", "", [{path, "/"}] ++ cookie_scheme(Req)),
-            {Code, Headers} = case couch_httpd:qs_value(Req, "fail", nil) of
-                nil ->
-                    {401, [Cookie]};
-                Redirect ->
-                    {302, [Cookie, {"Location", couch_httpd:absolute_uri(Req, Redirect)}]}
-            end,
-            send_json(Req, Code, Headers, {[{error, <<"unauthorized">>},{reason, <<"Name or password is incorrect.">>}]})
+            Cookie = mochiweb_cookies:cookie(
+                "AuthSession",
+                "",
+                [{path, "/"}] ++ cookie_scheme(Req)
+            ),
+            {Code, Headers} =
+                case couch_httpd:qs_value(Req, "fail", nil) of
+                    nil ->
+                        {401, [Cookie]};
+                    Redirect ->
+                        {302, [Cookie, {"Location", couch_httpd:absolute_uri(Req, Redirect)}]}
+                end,
+            send_json(
+                Req,
+                Code,
+                Headers,
+                {[{error, <<"unauthorized">>}, {reason, <<"Name or password is incorrect.">>}]}
+            )
     end;
 % get user info
 % GET /_session
-handle_session_req(#httpd{method='GET', user_ctx=UserCtx}=Req, _AuthModule) ->
+handle_session_req(#httpd{method = 'GET', user_ctx = UserCtx} = Req, _AuthModule) ->
     Name = UserCtx#user_ctx.name,
     ForceLogin = couch_httpd:qs_value(Req, "basic", "false"),
     case {Name, ForceLogin} of
         {null, "true"} ->
             throw({unauthorized, <<"Please login.">>});
         {Name, _} ->
-            send_json(Req, {[
-                % remove this ok
-                {ok, true},
-                {<<"userCtx">>, {[
-                    {name, Name},
-                    {roles, UserCtx#user_ctx.roles}
-                ]}},
-                {info, {[
-                    {authentication_handlers, [
-                       N || {N, _Fun} <- Req#httpd.authentication_handlers]}
-                ] ++ maybe_value(authenticated, UserCtx#user_ctx.handler, fun(Handler) ->
-                        Handler
-                    end) ++ maybe_value(authentication_db, config:get("chttpd_auth", "authentication_db"), fun(Val) ->
-                        ?l2b(Val)
-                    end)}}
-            ]})
+            send_json(
+                Req,
+                {[
+                    % remove this ok
+                    {ok, true},
+                    {<<"userCtx">>,
+                        {[
+                            {name, Name},
+                            {roles, UserCtx#user_ctx.roles}
+                        ]}},
+                    {info, {
+                        [
+                            {authentication_handlers, [
+                                N
+                             || {N, _Fun} <- Req#httpd.authentication_handlers
```

Comment:
-1 . I think I already mention it. I would put `||` at the same level as `N` on the previous line.